src/cascade/input_data/configuration/construct_country.py

```diff
@@ -1,19 +1,30 @@
 """
-Takes settings and creates a CovariateRecords object.
+Takes settings and creates a CovariateRecords object
 """
-from collections.__init__ import defaultdict
 
+from collections.__init__ import defaultdict, namedtuple
+
+import numpy as np
 import pandas as pd
 from scipy import spatial
+from scipy.interpolate import griddata
+
+import intervals as it
 
+from cascade.core import getLoggers
+from cascade.core.db import ezfuncs
 from cascade.input_data import InputDataError
 from cascade.input_data.db.country_covariates import country_covariates
 from cascade.input_data.db.demographics import get_all_age_spans
 from cascade.input_data.configuration.construct_study import CovariateRecords
-from cascade.core import getLoggers
 
 CODELOG, MATHLOG = getLoggers(__name__)
 
+FEMALE = -0.5
+MALE = 0.5
+BOTH = 0
+NEITHER = 0.1
```

Comment:
I know I called out NEITHER and we do need to support it (maybe? I'm still not sure what exactly the right thing is) but since we haven't agreed on how to represent it in the databases let's actually skip it for now. Sorry to waffle. At very least it can't be 0.1, we'll have to readjust how the whole system works to handle it I think.